hw/ip/otbn/doc/_index.md

```diff
@@ -370,6 +403,24 @@ The stack has a maximum depth of eight and the top of the stack is the current l
 
 ## Design Details
 
+### Random Numbers
+
+OTBN is connected to the [Entropy Distribution Network (EDN)]({{< relref "hw/ip/edn/doc" >}}) which can provide random numbers via the `RND` and `URND` CSRs and WSRs.
+
+`RND` provides bits taken directly from the EDN.
+As an EDN request can take time `RND` is backed by a single-entry cache containing the result of the most recent EDN request.
+A read from `RND` empties this cache.
+A prefetch into the cache, which can be used to hide the EDN latency, is triggered on any write to the `RND_PREFETCH` CSR.
```

Comment:
If I do two writes to RND_PREFETCH back to back, I assume the second has no effect? Maybe this needs rephrasing slightly?